### PR TITLE
Helper: Added timestamps to console output

### PR DESCRIPTION
**Pull request by 1480c1**
Added timestamps in the form of `HH:MM:SS $*`. Also bolded the ├ to match the ┌ and └.

### CODE REVIEW

**Comment by 1480c1** [comment]
The console will look like 
```
05:27:18 leptonica git.......................................[Up-to-date]
05:27:53   Running git update for tesseract...
05:28:01 ┌ tesseract git...................................[Updates found]
05:28:02 ├ Running autogen...
```

**Comment by wiiaboo** [comment]
Please make it optional and off by default. This is TMI for all future logs.

**Comment by 1480c1** [comment]
Okay. Should I ask a question of `do you want to see timestamps during compilation?` and add it to the ini? or just for only that session?

**Comment by 1480c1** [comment]
Ignore what I add for now, testing

**Comment by wiiaboo** [comment]
Make it a .ini option like the rest.

**Comment by 1480c1** [comment]
Okay

File: media-autobuild_suite.bat [hunk 1]
```diff
@@ -1170,6 +1170,30 @@ if %forceQuitBatchF%==2 set "forceQuitBatch=n"
 if %forceQuitBatchF% GTR 2 GOTO forceQuitBatch
 if %writeforceQuitBatch%==yes echo.forceQuitBatch=^%forceQuitBatchF%>>%ini%
 
+:logTime
+set "writelogTime=no"
+if %logTimeINI%==0 (
+    echo -------------------------------------------------------------------------------
+    echo -------------------------------------------------------------------------------
+    echo.
+    echo. Show timestamps of commands durring compilation?
```
Comment (wiiaboo) [inline media-autobuild_suite.bat]: during*
Same thing below.

Comment (1480c1) [reply media-autobuild_suite.bat]: oof, my english is bad.

**Comment by 1480c1** [comment]
It works now.

**Comment by 1480c1** [comment]
Now it is the it should output to the same length as without a timestamp.

File: build/media-suite_helper.sh [hunk 2]
```diff
@@ -23,20 +23,32 @@ fi
...
 
+if [[ $logTime = y ]]; then
+    datepad=12
+else
+    datepad=3
+fi
+
+do_print_time() {
+    if [[ $logTime = y ]]; then
+        echo -n "$(date +%H:%M:%S) "
```
Comment (wiiaboo) [inline build/media-suite_helper.sh]: Apparently, `printf` can also print the current time by itself without the need to call `date`.
`printf '%(%T)T %s\n' -1 stuff` or `printf '%(%H:%M:%S)T %s\n' -1 stuff`

Comment (wiiaboo) [reply build/media-suite_helper.sh]: Unfortunately it doesn't support milliseconds, afaict.

Comment (1480c1) [reply build/media-suite_helper.sh]: The more you learn

Comment (1480c1) [reply build/media-suite_helper.sh]: It works.
image
image

File: build/media-suite_helper.sh [hunk 3]
```diff
@@ -29,14 +29,23 @@ do_print_status() {
...
-        [[ ${1} =~ ^[a-zA-Z] ]] && echo "├ $*..." || echo -e "$*..."
+        if [[ $logTime = y ]]; then
+            [[ ${1} =~ ^[a-zA-Z] ]] && printf '%(%H:%M:%S)T %s\n' -1 \
+            "${bold}├${reset} $*..." \|| printf '%(%H:%M:%S)T %s\n' -1 "$*..."
+        else
+            [[ ${1} =~ ^[a-zA-Z] ]] && echo "${bold}├${reset} $*..." || echo -e "$*..."
+        fi
     else
         set_title "$* in $(get_first_subdir)"
         echo -e "${bold}$* in $(get_first_subdir)${reset}"
```
Comment (1480c1) [inline build/media-suite_helper.sh]: Do you suppose the same should be done for the non-logging option?

Comment (1480c1) [reply build/media-suite_helper.sh]: Only for the `Runnin * in *` and `* svn ... [Updated]`

Comment (wiiaboo) [reply build/media-suite_helper.sh]: Sure.

**Comment by 1480c1** [comment]
I think I am done for now, unless there is some other change or suggestion.
currently it will look like:
#### With Logging and timeStamp on
image
#### With Logging on and timeStamp off
image
#### With Logging off and timeStamp on
image
#### With Logging off and timeStamp off
image

**Comment by wiiaboo** [comment]
Great, thanks.

**Pull request merged**